independent-publisher-connectors/Fantasy Premier League/apiDefinition.swagger.json

```diff
@@ -0,0 +1,2184 @@
+                      "type": "boolean",
+                      "description": "has_cup"
+                    },
+                    "cup_league": {
+                      "type": "string",
+                      "description": "cup_league"
+                    },
+                    "rank": {
+                      "type": "string",
+                      "description": "rank"
+                    }
+                  },
+                  "description": "league"
+                },
+                "standings": {
+                  "type": "object",
+                  "properties": {
+                    "has_next": {
+                      "type": "boolean",
+                      "description": "has_next"
+                    },
+                    "page": {
+                      "type": "integer",
+                      "format": "int32",
+                      "description": "page"
+                    },
+                    "results": {
+                      "type": "array",
+                      "items": {
+                        "type": "object",
+                        "properties": {
+                          "id": {
+                            "type": "integer",
+                            "format": "int32",
+                            "description": "id"
+                          },
+                          "event_total": {
+                            "type": "integer",
+                            "format": "int32",
+                            "description": "event_total"
+                          },
+                          "player_name": {
+                            "type": "string",
+                            "description": "player_name"
+                          },
+                          "rank": {
+                            "type": "integer",
+                            "format": "int32",
+                            "description": "rank"
+                          },
+                          "last_rank": {
+                            "type": "integer",
+                            "format": "int32",
+                            "description": "last_rank"
+                          },
+                          "rank_sort": {
+                            "type": "integer",
+                            "format": "int32",
+                            "description": "rank_sort"
+                          },
+                          "total": {
+                            "type": "integer",
+                            "format": "int32",
+                            "description": "total"
+                          },
+                          "entry": {
+                            "type": "integer",
+                            "format": "int32",
+                            "description": "entry"
+                          },
+                          "entry_name": {
+                            "type": "string",
+                            "description": "entry_name"
```

Comment:
@Flow-Joe please make sure that descriptions look like valid sentences. You can refer to [coding standards](https://docs.microsoft.com/en-us/connectors/custom-connectors/coding-standards#definitions-and-parameters) for definition examples and to [English strings guidelines](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#create-quality-english-language-strings) for more information about strings.